include/exiv2/slice.hpp

```diff
@@ -153,22 +107,22 @@ namespace Exiv2
          *
          * - Must save data as a public member named `data_`.
          *
-         * - Must provide appropriate typedefs for iterator, const_iterator and
+         * - Must provide appropriate type aliases for iterator, const_iterator and
          *   value_type
          */
         template <template <typename data_type> class storage_type, typename data_type>
         struct ConstSliceBase : SliceBase
         {
-            typedef typename storage_type<data_type>::iterator iterator;
-            typedef typename storage_type<data_type>::const_iterator const_iterator;
-            typedef typename storage_type<data_type>::value_type value_type;
+            using iterator = typename storage_type<data_type>::iterator;
+            using const_iterator = typename storage_type<data_type>::const_iterator;
+            using value_type = typename storage_type<data_type>::value_type;
 
             /*!
              * Default contructor, requires begin to be smaller than end,
              * otherwise an exception is thrown. Also forwards all parameters to
              * the constructor of storage_
              */
-            ConstSliceBase(data_type& data, size_t begin, size_t end)
+            ConstSliceBase(data_type const& data, size_t begin, size_t end)
```

Comment:
This was a real change, only detected after ctor delegation.

Reply:
[minor] I think in the rest of the code we are using west-const.